devops/dev_api.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
 ./manage.py migrate --noinput
+./manage.py createinitialrevisions
```

Comment:
I'm not sure this is the correct place to put this, but note that it's non-destructive; if the revisions are already present it doesn't do anything. (This also applies to the same line in `prod_api.sh`.)